person.rb

```diff
@@ -1,4 +1,5 @@
-class Person
+require './nameable'
+class Person < Nameable
```

Comment:
- [ ] Please let's kindly make sure the Person class is correctly decorated by creating instances of the different classes and testing them if they are actually working. Try the code below to see if everything is working and submit it once more.

```
   person = Person.new(22, 'maximilianus')
      person.correct_name
      capitalizedPerson = CapitalizeDecorator.new(person)
      capitalizedPerson.correct_name
      capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
      capitalizedTrimmedPerson.correct_name
```